game/Enums.h

```diff
@@ -62,32 +62,8 @@ enum MoraleLevels
 	moraleRotten,				// 4
 };
 
-// Used by ScGroup.GetFirstOfType, ScGroup.UnitCount, and classes
-// derived from ScGroup which inherit these functions
-// (BuildingGroup, FightGroup, MiningGroup)
-enum UnitClassifactions			// [Note: **Typo** in name]
-{
-	clsAttack = 0,				// 0 (Lynx, Panther, Tiger, Scorpion)  [Not ESG, EMP, or Stickyfoam]
-	clsESG,						// 1 (Lynx, Panther, Tiger)
-	clsEMP,						// 2 (Lynx, Panther, Tiger)
-	clsStickyfoam,				// 3 (Lynx, Panther, Tiger)
-	clsSpider,					// 4
-	clsConvec,					// 5
-	clsRepairVehicle,			// 6
-	clsCargoTruck,				// 7
-	clsEarthworker,				// 8
-	clsColony,					// 9 (clsVehicle, not specified elsewhere) (RoboSurveyor, RoboMiner, GeoCon, Scout, RoboDozer, EvacuationTransport)
-	clsVehicleFactory,			// A
-	clsArachnidFactory,			// B
-	clsStructureFactory,		// C
-	clsOreMine,					// D (CommonOreMine, RareOreMine)
-	clsGuardPost,				// E
-	clsBuilding,				// F (more like non vehicle, and non other specified class) (does not include Arachnid Factory, includes beacons, disasters, Blast, Tube, pretty much any non vehicle?)
-	clsNotSet = 0x10,			// 10
-	clsAll = 0x11,				// 11 All vehicles and buildings
```

Comment:
Is there a reason we are explicitly setting clsNotSet and clsAll to 0x10 and 0x11? I was thinking maybe just removing the explicit set here since I don't think it is needed and info is in comments. Seeing it made me think maybe they were out of order or something.

Reply:
I think they were explicitly set because they're not actually real unit classes. They are pseudo classes. I don't know if there's a great reason for being explicit about only those 2 entries.

It might have been those two entries were explicitly set early on in development because the other entries weren't yet known. The code that was analysed to determine those values likely used certain values more commonly than others, such as the last two entries.